public/js/clients/firefox/commands.js

```diff
@@ -79,6 +83,12 @@ function removeBreakpoint(breakpointId) {
   return bpClient.remove();
 }
 
+function setBreakpointCondition(breakpointId, location, condition, noSliding) {
+  const bpClient = bpClients[breakpointId];
+  return bpClient.setCondition(threadClient, condition, noSliding)
+    .then((res) => onNewBreakpoint(location, [{}, res]))
```

Comment:
Also, can you rename `res` to `bpClient` or something to make it clear that it's just the breakpoint client?